preflight_check:

  topic_check:

    topics: [
      "estimation_manager/uav_state",
      "control_manager/diagnostics",
      "some_topic_that_does_not_exist",
    ]
